instancetypes/cx/1/cx1.yaml

```diff
@@ -16,6 +16,7 @@ metadata:
       In addition, in this series, the NUMA topology of the used
       cores is provided to the VM.
     instancetype.kubevirt.io/displayName: "Compute Exclusive"
+    instancetype.kubevirt.io/useCaseExamples: "Databases"
```

Comment:
The `Examples` seems redundant tbh, what about `instancetype.kubevirt.io/useCase` ?

Reply:
What about plural `instancetype.kubevirt.io/useCases`?